New ebuild for logfury

WPettersson opened the pull request:
Logfury is a boilerplate method-call logging library for python

gyakovlev commented on dev-python/logfury/logfury-0.1.2.ebuild:
```diff
@@ -0,0 +1,22 @@
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
```
I see `doc` useflag, but it's not used in any way)
Same goes for test, tests are not run.

WPettersson commented:
Updated commit message, and removed the unused USE flags. Both done by amending my original commit, I think that's the right way to go about this to keep history clean?

gyakovlev commented:
Yes, keeping history clean is important, you did it correctly, thanks!

I'll test it a bit later, otherwise looks good to me.

gyakovlev commented on dev-python/logfury/logfury-0.1.2.ebuild:
```diff
@@ -0,0 +1,21 @@
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	virtual/python-funcsigs[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}] "
```
non-conditional deps should be sorted  alphabetically.
Also remove the whitespace before the closing `"`

Also, RDEPEND is missing.
DEPEND != RDEPEND by default if EAPI=6
https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-650007.3.3

you should probably add six and funcsigs to RDEPEND,
and add `${RDEPEND}` to `DEPEND` along with setuptools.

check out for example dev-python/PyGithub/PyGithub-1.40_alpha4.ebuild how to do that. I just picked random ebuild that can hint you.

gyakovlev commented on dev-python/logfury/metadata.xml:
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
```
you are not a project =) change to `type="person"`

gyakovlev commented on dev-python/logfury/logfury-0.1.2.ebuild:
```diff
@@ -0,0 +1,21 @@
+HOMEPAGE="https://github.com/ppolewicz/logfury https://pypi.org/project/logfury/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
```
Nope, it's `BSD`.

project github says it's BSD 3-clause, 
https://github.com/ppolewicz/logfury/blob/master/LICENSE.txt
in gentoo it's called just `BSD`.

If you look into that file in licenses/BSD, it contains exactly 3 clauses =)

WPettersson commented:
Thanks for being patient with me. I've set myself as a person, not a project, fixed the license and fixed the DEPENDS/RDEPENDS. I'm not 100% sure on the whitespace, I couldn't find any style guidelines for ebuilds so I've tried to copy other ebuilds which means there is no empty line between the end of RDEPEND and the start of DEPEND.

gyakovlev commented:
Hey, no problem at all. That's a learning experience and you are doing great!

Here is the space I was referring to:
```
dev-python/six[${PYTHON_USEDEP}] "
                                ^ see that sucker before the final " ? =)
```



There should be an empty line between RDEPEND and DEPEND.

So last nit, add an empty line RDEPEND block, so there is a line before DEPEND starts and I'm going to merge it =)

WPettersson commented:
Ok, that should be it then.

gyakovlev commented:
Thanks!
I'll wait until ci bot comments and will merge it.
